test/Garnet.test/RespSetTest.cs

```diff
@@ -496,6 +497,88 @@ public void CanDoSPOPWithMoreCountThanSetSizeCommandLC()
             Assert.AreEqual(expectedResponse, strResponse);
         }
 
+        [Test]
+        public void CanDoSMOVECommandLC()
+        {
+            using var lightClientRequest = TestUtils.CreateRequest();
+
+            // source set
+            lightClientRequest.SendCommand("SADD \"mySourceSet\" \"oneS\"");
+            lightClientRequest.SendCommand("SADD \"mySourceSet\" \"twoS\"");
+            lightClientRequest.SendCommand("SADD \"mySourceSet\" \"threeS\"");
+            lightClientRequest.SendCommand("SADD \"mySourceSet\" \"fourS\"");
+            lightClientRequest.SendCommand("SADD \"mySourceSet\" \"common\"");
+
+            // destination set
+            lightClientRequest.SendCommand("SADD \"myDestinationSet\" \"oneS\"");
+            lightClientRequest.SendCommand("SADD \"myDestinationSet\" \"twoS\"");
+            lightClientRequest.SendCommand("SADD \"myDestinationSet\" \"threeS\"");
+            lightClientRequest.SendCommand("SADD \"myDestinationSet\" \"fourS\"");
+            lightClientRequest.SendCommand("SADD \"myDestinationSet\" \"common\"");
+
+            var expectedSuccessfulResponse = ":1\r\n";
+            var expectedFailureResponse = ":0\r\n";
+
+            // Successful move
+            var response = lightClientRequest.SendCommand("SMOVE \"mySourceSet\" \"myDestinationSet\" \"oneS\"");
```

Comment:
Probably good to also check that the correct value moved (i.e. verify contents of both sets)